lib/menu.ts

```diff
@@ -3,20 +3,23 @@
  * selected, and allows selection via the keyboard.
  *
  * The standard menu item offers enough flexibility to suffice for many needs, and may be replaced
- * entirely by a custom item. Most generally a menu item is a function that's given the observable
- * for whether the item is selected, and returns a DOM element.
+ * entirely by a custom item. For an item to be a selectable menu item, it needs `tabindex=-1`
+ * attribute set. If unset, the item will not be selectable.
+ *
+ * Further, if `dom.data('menuItemSelected', (yesNo: boolean) => {})` is set, that callback will be
+ * called whenever the item is selected and unselected, in addition to the item getting a
```

Comment:
instead of "in addition to...", maybe: "it may call onMenuItemSelected(yesNo) to keep the default behaivor of getting the suitable css class."